libs/hpc-core/.babelrc

```diff
@@ -1,3 +1,5 @@
 {
-  "presets": ["@nrwl/web/babel"]
+  "presets": [
+    "@nrwl/js/babel"
```

Comment:
Why did you keep `@nrwl/`? There are usages in other places too, so I won't comment everywhere. Search and replace them with `@nx/`